TwitterImagePipelineTests/TIPXWebPCodec.m

```diff
@@ -438,3 +427,75 @@ - (TIPImageContainer *)_tipx_renderImage
     }
     return [[TIPImageContainer alloc] initWithImage:image];
 }
+
+static BOOL TIPXWebPPictureImport(WebPPicture *picture, CGImageRef imageRef)
+{
+    __block vImage_Buffer convertedImageBuffer = {};
+
+    tipx_defer(^{
+        if (convertedImageBuffer.data) {
+            free(convertedImageBuffer.data);
+        }
+    });
+
+    if (!TIPXWebPCreateRGBADataForImage(imageRef, &convertedImageBuffer)) {
+        return NO;
+    }
+
+    return WebPPictureImportRGBA(picture, convertedImageBuffer.data, (int)convertedImageBuffer.rowBytes);
+}
+
+static BOOL TIPXWebPCreateRGBADataForImage(CGImageRef sourceImage, vImage_Buffer *convertedImageBuffer)
+{
+    if (convertedImageBuffer == NULL) {
+        return NO;
+    }
+
+    vImage_CGImageFormat sourceImageFormat = {
+        .bitsPerComponent = (uint32_t)CGImageGetBitsPerComponent(sourceImage),
+        .bitsPerPixel = (uint32_t)CGImageGetBitsPerPixel(sourceImage),
+        .colorSpace = CGImageGetColorSpace(sourceImage),
+        .bitmapInfo = CGImageGetBitmapInfo(sourceImage)
+    };
+
+    CGColorSpaceRef deviceRGBColorSpace = CGColorSpaceCreateDeviceRGB();
+    TIPXDeferRelease(deviceRGBColorSpace);
+
+    vImage_CGImageFormat convertedImageFormat = {
+        .bitsPerComponent = 8,
+        .bitsPerPixel = 32,
+        .colorSpace = deviceRGBColorSpace,
+        .bitmapInfo = kCGImageByteOrder32Big | kCGImageAlphaLast
```

Comment:
Yes - I believe the an alpha channel should be always included in the output (please correct me if I'm misunderstanding here). As I understand, if the source image has no alpha channel, `vImage` will just fill the alpha channel of the destination with fully opaque values.

I believe this shouldn't be a problem - I ran a few manual tests, and importing an image with all pixels fully opaque via `WebPPictureImportRGBA` will provides identical output to an image imported via `WebPPictureImportRGBX`, where alpha is ignored.